ibm/service/iampolicy/resource_ibm_iam_policy_template.go

```diff
@@ -274,12 +274,12 @@ func resourceIBMIAMPolicyTemplateCreate(context context.Context, d *schema.Resou
 	if err != nil {
 		return diag.FromErr(err)
 	}
-	userDetails, err := meta.(conns.ClientSession).BluemixUserDetails()
-	if err != nil {
-		return diag.FromErr(fmt.Errorf("failed to fetch BluemixUserDetails %s", err))
-	}
+	// userDetails, err := meta.(conns.ClientSession).BluemixUserDetails()
+	// if err != nil {
+	// 	return diag.FromErr(fmt.Errorf("failed to fetch BluemixUserDetails %s", err))
+	// }
 
-	accountID := userDetails.UserAccount
+	accountID := "e3aa0adff348470f803d4b6e53d625cf"
```

Comment:
I think accountID needs to be set back to `userDetails.UserAccount`
And is the commented out code leftover from testing?